### PR TITLE
FluentBit: parse the kubelet glog output

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
This makes FluentBit parse the kubelet logs when `collectSystemd` is enabled.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: config/logging/fluentbit/templates/configmap.yaml [hunk 1]
```diff
@@ -258,7 +278,5 @@ data:
     [PARSER]
         Name        glog
         Format      regex
-        Regex       ^(?<level>[A-Z])(?<time>[0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]+)\s+(?<thread>[0-9]+)\s+(?<file>[^:]+):(?<line>[0-9]+)\]\s*(?<message>.*)$
-        Time_Key    time
```
Comment (mrIncompetent) [inline config/logging/fluentbit/templates/configmap.yaml:262]: Taking the time from the log line breaks the forwarding to ES somehow.

Question is: Do we actually want to take the time from the log?
We already have a time key when the log comes through systemd/journald or containers.

Comment (xrstf) [reply config/logging/fluentbit/templates/configmap.yaml:262]: For containers I found that the time I could see in Kibana was "chunked" and seemingly it was the time when fluentbit shipped the log, not when the line was actually output. That was the reason for parsing the time and calling it `time`.

Comment (mrIncompetent) [reply config/logging/fluentbit/templates/configmap.yaml:262]: I'll see if i can do some testing on this tonight.
So far i only tested this with containerd, which has a different logging format than Docker :/

**Pull request merged**